core/src/test/scala/org/apache/spark/ExecutorAllocationManagerSuite.scala

```diff
@@ -1744,7 +1744,8 @@ class ExecutorAllocationManagerSuite extends SparkFunSuite {
       id: String,
       rp: ResourceProfile): Unit = {
     val cores = rp.getExecutorCores.getOrElse(1)
-    val execInfo = new ExecutorInfo("host1", cores, Map.empty, Map.empty, Map.empty, rp.id)
+    val execInfo = new ExecutorInfo("host1", cores, Map.empty, Map.empty, Map.empty, rp.id,
+      None, None)
```

Comment:
For the current PR state, we dont need changes to ExecutorAllocationManagerSuite, JsonProtocolSuite or  ExecutorMonitorSuite given the constructor ?

Reply:
That's a good point, let me back out the unrelated test changes (and that will also ensure the constructor change captured what was desired).